mirrord/cli/src/internal_proxy.rs

```diff
@@ -138,6 +146,11 @@ pub(crate) async fn proxy(args: InternalProxyArgs) -> Result<()> {
     // This makes the process not to receive signals from the "mirrord" process or it's parent
     // terminal fixes some side effects such as https://github.com/metalbear-co/mirrord/issues/1232
     nix::unistd::setsid().map_err(InternalProxyError::SetSidError)?;
```

Comment:
imo move all the "daemonizing" stuff to be done after printing the port for two reasons:
1. Have it at one place
2. Have it only after the proxy had launched successfully (so we might get errors out)

Besides that, I think we can remove the patch I added for not tracing on internal_proxy since now we can send traces it will just go nowhere (then when we use console or other thingy it'd also work)